script/brave_license_helper.py

```diff
@@ -28,6 +28,7 @@ def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
         os.path.join('brave', 'vendor', 'bat-native-ledger'),
         os.path.join('brave', 'vendor', 'brave-ios'),
         os.path.join('brave', 'vendor', 'brave_base'),
+        os.path.join('brave', 'vendor', 'test'),
```

Comment:
This is okay (and this is definitely not a blocker), but I'd suggest adding a comment above that line along the lines of "# Not an external component" since it doesn't quite fit under the existing comment.

Reply:
Added the suggested comment 👍